010-Getting-started/010-what-is-xata.mdx

```diff
@@ -1,300 +1,119 @@
+### Shared Postgres clusters
 
-If you just want to build apps and use the Xata serverless service, you **do not** need to read this document. We’re solving these problems so that you don’t have to.
+These are multi-tenant clusters that are shared between multiple customers. This is the default in the Xata service, and if you don't have a dedicated cluster, your databases will be automatically created on a shared cluster.
 
-Xata is a global multi-tenant system, composed of **cells**, each living in a region. At the high level, one Xata **cell** implements the following architecture:
+Shared clusters are cost-efficient and are the reason for which we can sustainably offer such a generous free-tier. The main billing metric for shared clusters is the storage size, with compute being bundled in. They come with rate limits and concurrency limits to prevent abuse. This means that they are typically cost efficient for applications with relatively small database sizes and low to moderate traffic.
 
-![Xata cell](/images/xata-cell.png)
+The shared clusters are limited when it comes to the PostgreSQL functionality that is enabled. For example, custom functions and triggers are not allowed, in order protect the security of the cluster.
 
-- The data is stored in PostgreSQL, which is considered the source of truth.
-- The PostgreSQL logical replication events are written in a Kafka queue, from where they can be read by multiple consumers.
-- One of these consumers indexes the data into Elasticsearch.
-- The Xata API layer offers a RESTful API offering both the functionality of the relational database (PostgreSQL) with the advanced search and analytics from the search engine/columnar store (Elasticsearch).
-- On top of this API, Xata has client SDKs and a web interface.
+Nevertheless, branches on shared clusters have zero cold starts, they are never paused, and come with high-availability and redundancy built-in, even on the free tier. The shared clusters are powered by AWS Aurora, which separates storage from compute and offers fast failover and point-in-time-recovery (PITR) backups.
 
-While the above architecture is a very common one in the industry, it hasn’t been —until now— productized in a service, which means a lot of teams were forced to either implement something similar to the above or live with reduced functionality.
+### Dedicated Postgres clusters
 
-## Going global
+<Alert status="warning">Dedicated clusters are currently in private alpha. Please reach out to request access.</Alert>
```

Comment:
Should the **request access** be linked to an email?

Reply:
yeah for now, I'd like to make this a TypeForm before launch

Reply:
I had problems putting the email a href in there, for some reason prettier didn't like it. And since we'll have a typeform soon anyway, I left alone it for now.